src/Types/ApplePayTypes.res

```diff
@@ -25,6 +25,13 @@ type shippingContact = {
 
 type paymentResult = {token: JSON.t, billingContact: JSON.t, shippingContact: JSON.t}
 type event = {validationURL: string, payment: paymentResult}
+type lineItem = {
+  label: string,
+  amount: string,
+  @optional \"type": string,
+}
```

Comment:
```suggestion
type lineItem = {
  label: string,
  amount: string,
  \"type"?: string,
}
```

is this optional type?? Because when I removed from code irt worked without any errors check once

Reply:
Strict type will also work, made the changes